src/tests/test_installations.py

```diff
@@ -15,26 +60,48 @@
 ]
 
 
-def _patch_session(monkeypatch, mock_table):
-    # DB patching
-    monkeypatch.setattr(installations, "installations", mock_table)
-    # Sterilize all DB interactions through CRUD override
-    monkeypatch.setattr(crud, "get", pytest.mock_get)
-    monkeypatch.setattr(crud, "fetch_all", pytest.mock_fetch_all)
-    monkeypatch.setattr(crud, "post", pytest.mock_post)
-    monkeypatch.setattr(crud, "put", pytest.mock_put)
-    monkeypatch.setattr(crud, "delete", pytest.mock_delete)
+DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%f"
+
+
+def update_only_datetime(entity_as_dict):
+    to_return = entity_as_dict.copy()
+    if "created_at" in to_return:
+        to_return["created_at"] = datetime.strptime(to_return["created_at"], DATETIME_FORMAT)
+    return to_return
+
+
+ACCESS_TABLE_FOR_DB = list(map(update_only_datetime, ACCESS_TABLE))
+USER_TABLE_FOR_DB = list(map(update_only_datetime, USER_TABLE))
+DEVICE_TABLE_FOR_DB = list(map(update_only_datetime, DEVICE_TABLE))
+EVENT_TABLE_FOR_DB = list(map(update_only_datetime, EVENT_TABLE))
+ALERT_TABLE_FOR_DB = list(map(update_only_datetime, ALERT_TABLE))
+MEDIA_TABLE_FOR_DB = list(map(update_only_datetime, MEDIA_TABLE))
+SITE_TABLE_FOR_DB = list(map(update_only_datetime, SITE_TABLE))
+INSTALLATION_TABLE_FOR_DB = list(map(update_only_datetime, INSTALLATION_TABLE))
 
 
-def test_get_installation(test_app, monkeypatch):
+async def init_test_db(monkeypatch, test_db):
+    monkeypatch.setattr(crud, "database", test_db)
+    await populate_db(test_db, db.accesses, ACCESS_TABLE_FOR_DB)
+    await populate_db(test_db, db.users, USER_TABLE_FOR_DB)
+
+    await populate_db(test_db, db.devices, DEVICE_TABLE_FOR_DB)
+    await populate_db(test_db, db.events, EVENT_TABLE_FOR_DB)
+    await populate_db(test_db, db.alerts, ALERT_TABLE_FOR_DB)
+    await populate_db(test_db, db.media, MEDIA_TABLE_FOR_DB)
+    await populate_db(test_db, db.sites, SITE_TABLE_FOR_DB)
+    await populate_db(test_db, db.installations, INSTALLATION_TABLE_FOR_DB)
```

Comment:
most of those patches are not required for most routes

Reply:
Those are not patches per se, but because of the foreign keys constraints, I can't populate a DB referring another DB if the foreign entities don't exist. And so on, hence why I need to populate everything here.